### DISCUSSION TITLE
ImportError: cannot import name 'warn_or_raise' from 'torch_geometric.data.data'

### DISCUSSION THREAD

**Discussion by frankdadale** [pyg-team/pytorch_geometric#6822] [Q&A]
I want to utilize the package torch_geometric.explain so upgrade to torch_geometic 2.2.0. 
However, when I import like this
​​``` python
from torch_geometric.explain import visualize_graph, visualize_feature_importance
​```
The following errors are reported
​``` python
~/anaconda3/lib/python3.8/site-packages/torch_geometric/explain/__init__.py in <module>
      1 from .config import ExplainerConfig, ModelConfig, ThresholdConfig
----> 2 from .explanation import Explanation
      3 from .algorithm import *  # noqa
      4 from .explainer import Explainer
      5 

~/anaconda3/lib/python3.8/site-packages/torch_geometric/explain/explanation.py in <module>
      4 from torch import Tensor
      5 
----> 6 from torch_geometric.data.data import Data, warn_or_raise
      7 
      8 

ImportError: cannot import name 'warn_or_raise' from 'torch_geometric.data.data' (/home/bmw_lab/anaconda3/lib/python3.8/site-packages/torch_geometric/data/data.py)
​```

**Comment by rusty1s** [comment]
What does `print(torch_geometric.__version__)` return? The `warn_or_raise` function should be available in PyG 2.2, see https://github.com/pyg-team/pytorch_geometric/blob/2.2.0/torch_geometric/data/data.py#L941.
